### PR TITLE
Relax rule about usage of non-WebSocket communication

### PR DESCRIPTION
**Pull request by Bluenix2**
As discussed internally, and later announced in `#code-jam-announcements`, this PR updates the wording of the rule to reflect how strict it is now enforced.

### CODE REVIEW

File: pydis_site/templates/events/pages/code-jams/9/rules.html [hunk 1]
```diff
@@ -11,19 +11,17 @@
...
     <li>
-        <p><strong>Your solution must only communicate through WebSockets</strong>. Any communication between the client and server (or any two server workers), must utilize WebSockets.</p>
-
         <p>
-            An exception to this rule is that communication with databases and files is allowed for accessing resources or for storage purposes.
-            For example, you may use PostgreSQL as a database but not its <code>NOTIFY</code> command.
-            Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also allowed.
+            <strong>The core of your project must use WebSockets as its communication protocol.</strong>.
+            This means that you are allowed to use other methods of communication where WebSockets cannot be implemented, however, that should be a non-significant portion of your project.
+            For example, serving static files for a website cannot be done over WebSockets and it does not pose as a significant portion of a project.
```
Comment (wookie184) [inline pydis_site/templates/events/pages/code-jams/9/rules.html]:
```suggestion
            For example, serving static files for a website cannot be done over WebSockets and it does not pose as a significant portion of a project, so it is allowed.
```
Just to make it crystal clear

File: pydis_site/templates/events/pages/code-jams/9/rules.html [hunk 2]
```diff
@@ -11,19 +11,17 @@
...
-    <li><p>Your solution must use one of the approved frameworks (a list will be released soon). It is not permitted to circumvent this rule by e.g. using the approved framework as a wrapper for another framework.</p></li>
+    <li><p>Your solution must use one of the approved frameworks. It is not permitted to circumvent this rule by e.g. using the approved framework as a wrapper for another framework.</p></li>
     <li>
-        <p><strong>Your solution must only communicate through WebSockets</strong>. Any communication between the client and server (or any two server workers), must utilize WebSockets.</p>
-
         <p>
-            An exception to this rule is that communication with databases and files is allowed for accessing resources or for storage purposes.
-            For example, you may use PostgreSQL as a database but not its <code>NOTIFY</code> command.
-            Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also allowed.
+            <strong>The core of your project must use WebSockets as its communication protocol.</strong>.
```
Comment (wookie184) [inline pydis_site/templates/events/pages/code-jams/9/rules.html:17]: I like this wording :+1:

File: pydis_site/templates/events/pages/code-jams/9/rules.html [hunk 3]
```diff
@@ -11,19 +11,17 @@
...
+            This means that you are allowed to use other methods of communication where WebSockets cannot be implemented, however, that should be a non-significant portion of your project.
+            For example, serving static files for a website cannot be done over WebSockets and it does not pose as a significant portion of a project.
         </p>
 
-        <p>Another exception is made for serving static files, which normally uses HTTP. You are allowed to serve a JavaScript-implemented client for your WebSocket server.</p>
+        <p>This rule does not apply to databases and files when used for <i>storage purposes</i> even though that may be a significant portion of your project. Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also exempted of this rule.</p>
 
-        <p>If you're interested in utilizing a particular non-WebSocket method of communication, reach out to the Events Team for discussion and approval</p>
+        <p>Reach out to the Events Team for advice if you're about your use of non-WebSocket communication.</p>
```
Comment (wookie184) [inline pydis_site/templates/events/pages/code-jams/9/rules.html]: Seem to be a few words that shouldn't be there :P
```suggestion
        <p>If you're unsure about your use of non-WebSocket communication, please reach out to the events team.</p>
```

Having it as a statement feels a little strange, but then again I wouldn't want people to hesitate to get in touch so I'd also be fine with your wording but fixed.

File: pydis_site/templates/events/pages/code-jams/9/rules.html [hunk 4]
```diff
@@ -11,19 +11,17 @@
...
-            Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also allowed.
+            <strong>The core of your project must use WebSockets as its communication protocol.</strong>.
+            This means that you are allowed to use other methods of communication where WebSockets cannot be implemented, however, that should be a non-significant portion of your project.
+            For example, serving static files for a website cannot be done over WebSockets and it does not pose as a significant portion of a project.
         </p>
 
-        <p>Another exception is made for serving static files, which normally uses HTTP. You are allowed to serve a JavaScript-implemented client for your WebSocket server.</p>
+        <p>This rule does not apply to databases and files when used for <i>storage purposes</i> even though that may be a significant portion of your project. Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also exempted of this rule.</p>
```
Comment (wookie184) [inline pydis_site/templates/events/pages/code-jams/9/rules.html]:
```suggestion
        <p>This rule does not apply to databases and files when used for <i>storage purposes</i> even though that may be a significant portion of your project. Working with subprocesses (through stdin/stdout or <code>multiprocessing.Pool()</code>/<code>concurrent.futures.ProcessPoolExecutor()</code>) is also exempt from this rule.</p>
```

**Review by wookie184 — CHANGES_REQUESTED** [review]
Thanks

File: pydis_site/templates/events/pages/code-jams/9/rules.html (hunk 1 above)
Comment (Bluenix2) [reply pydis_site/templates/events/pages/code-jams/9/rules.html]: I am switching this out to "therefore"

File: pydis_site/templates/events/pages/code-jams/9/rules.html (hunk 3 above)
Comment (Bluenix2) [reply pydis_site/templates/events/pages/code-jams/9/rules.html]: I see now, I must have stared at this for so long that it didn't occur to me.

**Review by jchristgit — APPROVED** [review]

**Review by wookie184 — APPROVED** [review]

**Pull request merged**